modules/db_dashboard.py

```diff
@@ -451,7 +455,7 @@ def load_assignment_metrics(self, username, studentView=False):
                                                "due_date":assign["duedate"].date().strftime("%m-%d-%Y")}
 
 # This whole object is a workaround because these strings
-# are not generated and stored in the db. This needs automating
+# are not generated and stored in the current.db. This needs automating
```

Comment:
A bit of overzealous find-and-replace...

Reply:
This `IdConverter` object requires further investigation... I'm pretty sure its not used, and is insanely incomplete if it is, in fact thethings it is looking for are in the database.